package.json

```diff
@@ -88,6 +88,7 @@
   },
   "dependencies": {
     "@types/json-schema": "^7.0.7",
-    "json-pointer": "0.6.2"
+    "json-pointer": "0.6.2",
+    "randexp": "^0.5.3"
```

Comment:
This is strange to include this as dependency but not use it directly. I think it's better to add 2 other dependencies here.

But I'm also concerned by those dependencies. They are not very active. Last update 5 years ago.